lib/improver/tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -789,12 +789,10 @@ def test_specific_cycletime(self):
         cube1 = self.cube[0].copy()
         model_crd1 = iris.coords.DimCoord([0], long_name='model', units=1)
         cube1.add_aux_coord(model_crd1)
-        cube1 = iris.util.new_axis(cube1, scalar_coord=model_crd1)
```

Comment:
This is the only change I didn't quite grasp as I could not see a change in MergeCubes that made this redundant. Was it always redundant or have I overlooked something?

Reply:
`MergeCubes._equalise_cube_coords` used to slice over any mismatched dimension coordinates, demoting them to scalar so that they could be merged over.  That use case wasn't really justifiable, as we don't expect to have scalar dimensions on our input cubes, and anything where the dimensions aren't scalar ought to be concatenated rather than merged.  Having removed that function, promoting scalar coordinates to dimension coordinates for no reason (worth noting these never would be dimensional - they're created as scalar) threw an error!  So I changed the test.

Reply:
Oh of course. I rather ignored the massive deleted section as we had discussed it yesterday. Thanks.